xml/System/DateTime.xml

```diff
@@ -4236,130 +4236,79 @@ juillet 2009
+
+-   A string with a time but no date component. The method assumes the current date unless you call the <xref:System.DateTime.Parse%28System.String%2CSystem.IFormatProvider%2CSystem.Globalization.DateTimeStyles%29> overload and include <xref:System.Globalization.DateTimeStyles.NoCurrentDateDefault?displayProperty=nameWithType> in the `styles` argument, in which case the method assumes a date of January 1, 0001.  
+
+- A string with a time component that includes only the hour and an AM/PM designator, with no date component. The method assumes the current date and a time with no minutes and no seconds. You can change this behavior by calling the <xref:System.DateTime.Parse%28System.String%2CSystem.IFormatProvider%2CSystem.Globalization.DateTimeStyles%29> overload and include <xref:System.Globalization.DateTimeStyles.NoCurrentDateDefault?displayProperty=nameWithType> in the `styles` argument, in which case the method assumes a date of January 1, 0001.
+
+-   A string that includes time zone information and conforms to ISO 8601. In the following examples, the first string designates Coordinated Universal Time (UTC), and the second designates the time in a time zone that's seven hours earlier than UTC:  
   
-     2008-11-01T19:35:00.0000000Z   
-    2008-11-01T19:35:00.0000000-07:00  
+    "2008-11-01T19:35:00.0000000Z"   
+    "2008-11-01T19:35:00.0000000-07:00"  
   
 -   A string that includes the GMT designator and conforms to the RFC 1123 time format; for example:  
   
-     Sat, 01 Nov 2008 19:35:00 GMT  
+     "Sat, 01 Nov 2008 19:35:00 GMT"  
   
 -   A string that includes the date and time along with time zone offset information; for example:  
   
-     03/01/2009 05:42:00 -5:00  
+     "03/01/2009 05:42:00 -5:00"  
+
+The following example parses strings in each of these formats by using the formatting conventions of the current culture, which in this case is the en-US culture:
+
+[!code-csharp-interactive[Default parsing](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.DateTime.Parse/cs/Parse6.cs)]
+[!code-vb[Default parsing](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Parse/vb/Parse6.vb)]  
   
- If `s` is the string representation of a leap day in a leap year in the calendar used by the parsing method (see [Parsing and cultural conventions](#Culture)), the <xref:System.DateTime.Parse%2A> method parses `s` successfully. If `s` is the string representation of a leap day in a non-leap year, the method throws a <xref:System.FormatException> exception.  
+ If `s` is the string representation of a leap day in a leap year in the calendar used by the parsing method (see [Parsing and cultural conventions](#Culture)), the <xref:System.DateTime.Parse%2A> method parses `s` successfully. If `s` is the string representation of a leap day in a non-leap year, the method throws a <xref:System.FormatException>.  
```

Comment:
same comment on variable `s`: it hasn't been introduced. Since it is used in the samples below, it may be best to introduce it here.